Source/FLHook/HkCbIServerImpl.cpp

```diff
@@ -1017,50 +993,8 @@ namespace HkIServerImpl
 				return;
 			}
 
-			CALL_PLUGINS_V(PLUGIN_HkIServerImpl_Login, __stdcall, (struct SLoginInfo const &li, unsigned int iClientID), (li, iClientID));
-
-
-			// check for ip ban
-			wstring wscIP;
-			HkGetPlayerIP(iClientID, wscIP);
-
-			foreach(set_lstBans, wstring, itb)
```

Comment:
we seem to be getting rid of random things without proper cleanup here - set_lstBans is still defined somewhere